conan/tools/build/flags.py

```diff
@@ -203,11 +203,11 @@ def cppstd_msvc_flag(visual_version, cppstd):
     if visual_version >= "193":
         v23 = "c++latest"
 
-    return {"14": v14, "17": v17, "20": v20, "23": v23}.get(str(cppstd), None)
+    return {"14": v14, "17": v17, "20": v20, "23": v23}.get(str(cppstd))
 
 def _cppstd_msvc(visual_version, cppstd):
     flag = cppstd_msvc_flag(visual_version, cppstd)
-    return "/std:%s" % flag if flag else None
+    return "/std:%s" % flag
```

Comment:
This one might might return a literal string ``/std:None`` which is undesired. Which is different than returning ``None`` (the priority seems to be first the string interpolation, then the if, so ``return None`` if flag is not defined

Reply:
Thanks for the quick feedback, trying with small steps. The unittests did a fine job ;)